packages/markdown/remark/src/index.ts

```diff
@@ -49,13 +56,19 @@ export async function renderMarkdown(
 		.use(remarkInitializeAstroData)
 		.use(isAstroFlavoredMd ? [remarkMdxish, remarkMarkAndUnravel, remarkUnwrap, remarkEscape] : []);
 
-	if (remarkPlugins.length === 0 && rehypePlugins.length === 0) {
+	if (Array.isArray(remarkPlugins) && remarkPlugins.length === 0) {
 		remarkPlugins = [...DEFAULT_REMARK_PLUGINS];
-		rehypePlugins = [...DEFAULT_REHYPE_PLUGINS];
+	}
+	if (Array.isArray(rehypePlugins) && rehypePlugins.length === 0) {
+		remarkPlugins = [...DEFAULT_REHYPE_PLUGINS];
```

Comment:
I discovered that, if you have `rehypePlugins: []` but _not_ `remarkPlugins: []`, **our default remark plugins are removed.** This should fix that!